net/adblock/files/README.md

```diff
@@ -269,7 +269,9 @@ Finally enable email support and add a valid email address in LuCI.
 [...]
 ```
   
-**Cronjob for a regular block list update (`/etc/crontabs/root`):**
+**Cronjob for regular block list updates (`/etc/crontabs/root`):**
+
+The following command as a cron job updates each individual block list from their source so that they hold the most current IPs/domains to block:
```

Comment:
Please Remove "IPs/" ... it's solely domain based.

Reply:
> Please Remove "IPs/" ... it's solely domain based.

No, because I could not find the possibility to do so here on GitHub. :-(